### DISCUSSION TITLE
Format Datetime To Hide D/M/YYYY

### DISCUSSION THREAD

**Discussion by choandrew** [perspective-dev/perspective#1442] [Q&A]
Hello,

So the default format for a datetime cell is:
DD/MM/YYYY, HH:MM:SS AM/PM.

Is there a way to hide the DD/MM/YYYY and make it military time?

**Comment by dhirschfeld** [comment]
I've arrived here looking for any way to specify the formatting of datetimes in the JupyterLab widget. Is it possible?
